platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -936,8 +946,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.  (Token replacement is only supported for literal
- `textField` values--not for property functions.)
+ like `{field_name}`. (`{token}` replacement is only supported for literal
+ `textField` values; not for property functions.)
```

Comment:
Note that this implies support for `{token}` replacement in expressions. That happens to be the case at the moment, but I’m not sure if it’s intentional. (Also, in the event that you or someone else happens to update this documentation in the style specification, the semicolon should be a comma.)

Reply:
mapbox/mapbox-gl-js#5959 mentions data expressions and fixes the typo.

Reply:
I think we should just remove references to token replacement entirely. We want people to migrate to expressions (and we're not going to support token replacement within them: #10399).

Reply:
The published style specification is the canonical documentation for runtime styling for both GL JS (which supports expressions) and Android (which doesn’t yet in v5.3.0). @tobrun, do you think it would be safe to remove the bit about the `{token}` syntax now, before the next Android release which has expression support?